src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
Sidebar: project's a bit of an overloaded term in MSBuild world. So, to clarify, translate these terms accordingly:

makefile.shade => 'build project'
project.json => 'code project'


The intent of this target is to be agnostic about whether it runs in a build or code project. That said, the vision for now is to only use this in a build project.

A WIP version of that vision: https://github.com/aspnet/DotNetTools/blob/58d429609db3f4f594f03e0023dce36b99182fb8/build.xml. The goal I'm working towards an MSBuild version of KoreBuild that has many of the same features of KoreBuild. i.e. each repo will only need to define a minimal makefile that extends a default lifecycle.